templates/customgroup.tpl

```diff
@@ -0,0 +1,18 @@
+{* FILE: customfieldnames/templates/customgroupfield.tpl to add custom field for custom data set*}
+<table id="custom-fields" class="form-layout">
+  <tbody>
+    <tr id="table_name-tr">
+      <td class="label">{$form.table_name.label}</td>
+      <td class="html-adjust">{$form.table_name.html}</td>
+    </tr>
+
+    <tr id="name-tr">
+      <td class="label">{$form.name.label}</td>
+      <td class="html-adjust">{$form.name.html}</td>
+    </tr>
+  </tbody>
+</table>
+
+<script type="text/javascript">
+  cj('#custom-fields').insertAfter('div.crm-submit-buttons:first');
+</script>
```

Comment:
currently in core the field that collects the label is called "Set Name" -- which is misleading and confusing with what we're doing. let's relabel it with some jquery to be "Set Label" (I may suggest the change in core as well, but for now, let's just modify it in this extension)

Reply:
actually, on second thought -- rather than do it with js, we can make that label change in build form. you'll need to get the element as a reference and then change the value. something like:

```
if ($form->elementExists('title') {
  $ele =& $form->getElement('title');
  //then find out where the title is set in that object and change it
}
```

Reply:
I have changed the label of title field to "Set Label" using form hook